src/main/flight/mixer.c

```diff
@@ -227,6 +233,8 @@ const mixer_t mixers[] = {
     { 2, 1, mixerDualcopter },     // MIXER_DUALCOPTER
     { 1, 1, NULL },                // MIXER_SINGLECOPTER
     { 4, 0, mixerAtail4 },         // MIXER_ATAIL4
+    { 4, 1, mixerQuadX },          // MIXER_QUADX_TILT
+	{ 8, 1, mixerOctoX8 },         // MIXER_OCTOX_TILT
```

Comment:
fixed in 957c332

sorry but for other project i use tab, and sometimes i forgot to switch the style in the editor.